Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.py

```diff
@@ -661,6 +662,128 @@ def get_blacklist_endpoints():
     return_outputs(tableToMarkdown('CiscoISE Blacklist Endpoints', data, removeNull=True), context, endpoints)
 
 
+def get_endpoint_id_by_name(mac_address=None):
+    """
+    Returns endpoint id by specific mac address
+    Only compatible with Cisco ISE versions 2.3
+    """
+    if not is_mac_address(mac_address):
```

Comment:
consider using the `is_mac_address` utility function from commonserverpython

Reply:
i do not see a local definition for "is_mac_address" in this file so i am assuming its already from commonserverpython. Please let me know if i am wrong